packages/grafana-ui/src/components/Button/Button.mdx

```diff
@@ -25,7 +25,7 @@ Used for "call to action".
 
 ## Secondary
 
-The secondary button, used for "cancel" or aborting.
+The secondary `Button` can trigger various actions. When next to the primary `Button`, it can for example be used for "Cancel" or "Abort" actions. Apart from that, the secondary variant is the main `Button` variant in Grafana and is used for most buttons except when there is an important main action.
```

Comment:
The last sentence is a bit hard to understand. Maybe breaking the whole paragraph into two cases:
1.  When next to the primary `Button`, it can for example be used for "Cancel" or "Abort" actions.
2. When there is no main important action on a given page, the secondary button is used a default primary one.